### PR TITLE
fix deploy not using built template bug and not adding copy code when converting fun.yml to Funfile

### PR DESCRIPTION
**Pull request by tanhe123**

### CODE REVIEW

File: lib/build/parser.js [hunk 1]
```diff
@@ -82,6 +84,11 @@ function funymlToFunfile(funymlPath) {
       break;
     }
     case 'shell': {
+      if (firstShellCmd) {
```
Comment (vangie) [inline lib/build/parser.js]: 这里加一段注释说明一下为什么需要这么干吧

**Comment by tanhe123** [comment]
fix https://github.com/alibaba/funcraft/issues/448

**Comment by tanhe123** [comment]
fix https://github.com/alibaba/funcraft/issues/483

File: lib/build/parser.js (hunk 1 above)
Comment (tanhe123) [reply lib/build/parser.js]: done

**Pull request merged**